Developers/Unity Project/Spooksville/Assets/Scripts/Player/Inventory.cs

```diff
@@ -1,18 +1,168 @@
 ﻿using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using UnityEngine.UI;
 
 public abstract class Inventory
 {
     private static List<Weapon> weapons = new List<Weapon>();
-    
+
+    private static bool isPressed;
+
+    private static int window = 1;
+    private static int windows = 0;
+    private static int row;
+    public static int Row
+    {
+        get
+        {
+            return row;
+        }
+
+        set
+        {
+            if (value < 0)
+            {
+                row = 0;
+
+                if (window - 1 < 1)
+                {
+                    window = 1;
+                } else
+                {
+                    window -= 1;
+                }
+
+                return;
+            }
+
+            if (value > 2)
+            {
+                row = 2;
+
+                if (window + 1 > windows)
+                {
+                    window = windows;
+                }
+                else
+                {
+                    window += 1;
```

Comment:
Another way you could do this sort of thing is turn `window` into a getter and just recompute the value every time, if it's always derived from `row`.